frontend/src/index.css

```diff
@@ -31,4 +31,4 @@ h1 {
     color: #213547;
     background-color: #ffffff;
   }
-}
+} */
```

Comment:
If you aren't using the CSS defined for `:root` you can just remove it. 
It doesn't do anything special by being declared